### PR TITLE
Add images to ArtistCollectionRail

### PR DESCRIPTION
**Pull request by eessex**
- Fetches top 3 merchandisable artworks and includes images in artworks rail
- Displays 2 or 1 images if fewer than 3 are returned
- Displays headerImage as fallback if no artworks are returned

Screen Shot 2019-03-12 at 4 39 18 PM

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 2aebc862dd7fdd830c5c208cd46456c7ab1fd8bd

https://deploy-preview-2152--artsy-reaction.netlify.com

File: src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx [hunk 1]
```diff
@@ -26,8 +27,17 @@ export class ArtistCollectionEntity extends React.Component<CollectionProps> {
...
-    const { price_guidance, slug, title } = this.props.collection
+    const {
+      artworks: { hits },
+      headerImage,
+      price_guidance,
+      slug,
+      title,
+    } = this.props.collection
     const formattedTitle = (title && title.split(": ")[1]) || title
+    const bgImages = map(hits, "image.url")
```
Comment (xtina-starr) [inline src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx]: what does passing the `"image.url"` string do exactly?

File: src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx [hunk 2]
```diff
@@ -68,13 +86,12 @@ export const StyledLink = styled(Link)`
...
-// TODO: add background url when supported by Kaws
-const ImgPlaceholder = styled.div`
-  width: 85px;
+export const ArtworkImage = styled.img<{ width: number }>`
+  width: ${({ width }) => width}px;
   height: 125px;
   background-color: ${color("black10")};
-  background-position: center;
-  background-size: cover;
+  object-fit: cover;
```
Comment (xtina-starr) [inline src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx]: nice!

**Comment by xtina-starr** [comment]
This looks good to me. Feel free to merge when you are ready.

File: src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx (hunk 1 above)
Comment (eessex) [reply src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx]: The string is used to get a nested value of the object. The structure of the artwork is `{ ...artwork, image: { url: "string" } }`, by using[ lodash's `map`](https://lodash.com/docs/4.17.11#map) we can grab the values from an array of objects. This is the same as `pluck` in underscore.

File: src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx (hunk 2 above)
Comment (eessex) [reply src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx]: Changed this from a background image to an actual image, based on a suggestion from @joeyAghion that including image `alt` could be an SEO win.

**Comment by artsyit** [comment]
:rocket: PR was released in v15.3.5 :rocket:

**Pull request merged**